playbooks/robusta_playbooks/babysitter.py

```diff
@@ -10,7 +10,11 @@
 from robusta.api import *
 
 
-class BabysitterConfig(BaseModel):
+class BabysitterConfig(ActionParams):
+    """
+    :var fields_to_monitor: List of yaml attributes to monitor. Matching is done using 'contains' operator.
```

Comment:
I don't understand what you mean re. the contains operator

Reply:
fixed